unstoppable-ios-app/domains-manager-ios/Modules/DomainsList/Cells/DomainsCollectionListCell/DomainsCollectionListCell.swift

```diff
@@ -57,6 +57,9 @@ extension DomainsCollectionListCell {
         case .deprecated(let tld):
             statusMessage.setComponent(.deprecated(tld: tld))
             statusMessage.isHidden = false
+        case .newNonInteractable:
+            //statusMessage.setComponent() // TODO: Oleg plz confirm
```

Comment:
@rommex This is where we set a running line (deprecated, updating records, minting in progress, etc.) Is there a specific UI for ENS domains? If no - status message should be hidden